uaclient/fix.py

```diff
@@ -0,0 +1,720 @@
+import enum
```

Comment:
Since the code in this file is only used by the api, and the data structures defined here _are_ the API (and can't be changed without breaking compat), should this code live in the `api.u.pro.security.fix` module?

Reply:
That is fair, I will move the entire module to this path